### PR TITLE
Rely on the bundled binaries for all languages

### PR DESCRIPTION
**Pull request by fundthmcalculus**
Closes https://github.com/trinsic-id/server/issues/1279

### CODE REVIEW

File: .github/install-okapi-binaries-action/action.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+name: "Install Okapi Binaries"
+description: "Install Okapi binaries on the github action runner"
+
+runs:
+  using: "composite"
+  steps:
+    - name: Download Libs
+      run: |
+        curl -L -o libs.zip https://github.com/trinsic-id/okapi/releases/download/v1.6.0/libs.zip
```
Comment (janpieterz) [inline .github/install-okapi-binaries-action/action.yaml]: Should the version be hard-coded? 🤔  Could we somehow curl the last release number, or retrieve it from. our local git clone?

**Review by janpieterz — APPROVED** [review]

File: .github/install-okapi-binaries-action/action.yaml (hunk 1 above)
Comment (fundthmcalculus) [reply .github/install-okapi-binaries-action/action.yaml]: Okapi and SDK aren't necessarily in lockstep, so we can't use the local version number. I'll see if I can curl the latest released version (I know I can, question is what's the easiest way in powershell XD)

Comment (fundthmcalculus) [reply .github/install-okapi-binaries-action/action.yaml]: Hard part of this is the macos behavior is different. Let me see what I can do about that.

Comment (fundthmcalculus) [reply .github/install-okapi-binaries-action/action.yaml]: Got it working - finally!

**Pull request merged**